armi/operators/tests/test_operators.py

```diff
@@ -543,32 +543,44 @@ def test_getMaxBurnSteps(self):
 
 
 class TestInterfaceAndEventHeaders(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.o, cls.r = test_reactors.loadTestReactor(
+            inputFileName="smallestTestReactor/armiRunSmallest.yaml",
+            customSettings={CONF_TIGHT_COUPLING: True},
+        )
+        cls.r.p.cycle = 0
+        cls.r.p.timeNode = 1
+        cls.r.core.p.coupledIteration = 7
+
     def test_expandCycleAndTimeNodeArgs_Empty(self):
         """When *args are empty, cycleNodeInfo should be an empty string."""
```

Comment:
The docstring for each of the test methods in this test class need to be updated.

Reply:
OOPS I forgot (I had noticed it)!

Reply:
9f8ee4a45c1940b0fa3935ee5e139c5325c8a955